tools/watermill-cli/cmd/consume.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"os"
+	"time"
+
+	"github.com/ThreeDotsLabs/watermill/message/infrastructure/io"
+	"github.com/ThreeDotsLabs/watermill/message/router/middleware"
+	"github.com/ThreeDotsLabs/watermill/message/router/plugin"
+	"github.com/pkg/errors"
+	"github.com/spf13/viper"
+
+	"github.com/ThreeDotsLabs/watermill/message"
+
+	"github.com/spf13/cobra"
+)
+
+// consumer is initialized by the parent command to the pub/sub provider of choice.
+var consumer message.Subscriber
+
+func addConsumeCmd(parent *cobra.Command, topicKey string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "consume",
+		Short: "Consume messages from a pub/sub and print them to stdout",
+		Long: `Consume messages from the pub/sub of your choice and print them on the standard output.
+
+For the configuration of particular pub/sub providers, see the help for the provider commands.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			topic := viper.GetString(topicKey)
+			router, err := message.NewRouter(
+				message.RouterConfig{
+					CloseTimeout: 5 * time.Second,
+				},
+				logger,
+			)
+			if err != nil {
+				return errors.Wrap(err, "could not create router")
+			}
+
+			router.AddMiddleware(middleware.InstantAck)
```

Comment:
shouldn't you remove it? :poultry_leg:

Reply:
Good call, I removed it in the other router, missed this one

Reply:
b3ae9f252ccb6e90efcf9f72512765a9df71cff4